docs/pages/tutorial/eas/multiple-app-variants.mdx

```diff
@@ -182,7 +202,7 @@ Now, our development build will run on both Android and iOS, displaying the modi
   name="EAS_TUTORIAL"
   summary={
     <>
-      We successfully switched to <strong>app.config.js</strong> for dynamic settings, added
+      We successfully created <strong>app.config.js</strong> for dynamic configuration, added
```

Comment:
```suggestion
      We successfully created <strong>app.config.js</strong> just for our dynamic configuration (while leaving **app.json** for configuration that will never change), added
```
Trying to add a little context about what goes in app.config.js vs app.json (maybe there's better wording for that)